<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFFCSMSDEvaluationFitMSDDialog</class>
 <widget class="QDialog" name="QFFCSMSDEvaluationFitMSDDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fit MSD power law</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0">
   <item>
    <widget class="QFPlotter" name="pltDistribution" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DataCutSliders" name="datacut" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; 〈r&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;(τ)〉 = &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QFDoubleEdit" name="edtPre"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string> * </string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string> * &amp;tau; ^</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="chkFixD">
       <property name="text">
        <string>fix</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QCheckBox" name="chkFixAlpha">
       <property name="text">
        <string>fix</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QFDoubleEdit" name="edtD"/>
     </item>
     <item row="1" column="7" rowspan="3">
      <widget class="QPushButton" name="btnFit">
       <property name="text">
        <string>&amp;Fit</string>
       </property>
       <property name="icon">
        <iconset resource="fcsmsdevaluation.qrc">
         <normaloff>:/fcsmsdevaluation/fit_fitcurrent.png</normaloff>:/fcsmsdevaluation/fit_fitcurrent.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>D</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>prefactor:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QFDoubleEdit" name="edtAlpha"/>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>alpha</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>fit type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="6">
      <widget class="QComboBox" name="cmbFitType"/>
     </item>
     <item row="2" column="6">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFPlotter</class>
   <extends>QWidget</extends>
   <header>qfplotter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QFDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qfdoubleedit.h</header>
  </customwidget>
  <customwidget>
   <class>DataCutSliders</class>
   <extends>QWidget</extends>
   <header>datacutslider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="fcsmsdevaluation.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFFCSMSDEvaluationFitMSDDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QFFCSMSDEvaluationFitMSDDialog</receiver>
   <slot>saveResults()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>helpRequested()</signal>
   <receiver>QFFCSMSDEvaluationFitMSDDialog</receiver>
   <slot>showHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveResults()</slot>
  <slot>showHelp()</slot>
 </slots>
</ui>
